scripts/ai/AIDriveStrategyCourse.lua

```diff
@@ -625,10 +625,15 @@ function AIDriveStrategyCourse:isCloseToCourseStart(distance)
     return self.course:getDistanceFromFirstWaypoint(self.ppc:getCurrentWaypointIx()) < distance
 end
 
---- Event raised when the driver has finished.
---- This gets called in the :stopCurrentAIJob(), as the giants code might stop the driver and not the active strategy.
-function AIDriveStrategyCourse:onFinished()
-    self:raiseControllerEvent(self.onFinishedEvent)
+--- Event raised when the driver was stopped.
+---@param hasFinished boolean|nil flag passed by the info text
+function AIDriveStrategyCourse:onFinished(hasFinished)
+    self:raiseControllerEvent(self.onFinishedEvent, hasFinished)

```

Comment:
hasFinished is not used in any onFinished() controller function, or is this for the future only?